cypress/e2e/another-test.cy.js

```diff
@@ -0,0 +1,20 @@
+describe('SDEP-5657-2 Test Scenario', () => {
+  it('User clicks on the second link provided by Google search', () => {
+    // 1. User Opens Google page
+    cy.visit('https://www.google.com')
```

Comment:
Sometimes you and line with ';" and sometimes not, you need to choose one option. Add ";" for every line end or remove it